project/Dependencies.scala

```diff
@@ -2,8 +2,8 @@ import sbt._
 
 object Dependencies {
   val core = Seq(
-    Library.slick,
-    Library.slickHikariCP,
+    // Library.slick,
```

Comment:
I had to move everything regarding slick into the main build file because I wanted to differentiate the library import based on `scalaVersion.value`. I couldn't get it in this file.